cpp/ps0003/ps0003_Solution.cpp

```diff
@@ -1,40 +1,21 @@
-
-  /* help your brother check if the number is even or odd
-        - if the number is even, print "Number is even"
-        - if the number is odd, print "Number is odd"
-
-        - ex:
-          => 909
-          <= Number is odd
-
-        - another ex:
-          => 20000
-          <= Number is odd
-    */
-
-
 #include<iostream>
 
 int main() {
 
-  
-
-
-
+    // prompting user for number 2/2
     int number;
-
     std::cin >> number;
 
+    // checking if the number is odd or even 3/3
     if (number%2==0)
     {
         std::cout<<"Number is even"<< std::endl;
-
     }
     else
     {
         std::cout<<"Number is odd"<< std::endl;
     }
 
-
     return 0;
 }
+// great work 😎 8/10

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code

you can use _ternary operator_
```cpp
std::cout << "Number is " << ((x % 2 == 0) ? "even" : "odd") << std::endl
```